teku/src/main/java/tech/pegasys/teku/config/TekuConfiguration.java

```diff
@@ -202,6 +203,15 @@ public TekuConfiguration build() {
           eth2NetworkConfigurationBuilder.build();
       final Spec spec = eth2NetworkConfiguration.getSpec();
 
+      Eth1Address depositContractAddress = eth2NetworkConfiguration.getEth1DepositContractAddress();
+      storageConfigurationBuilder.eth1DepositContractDefault(depositContractAddress);
+      powchainConfigBuilder.depositContractDefault(depositContractAddress);
+      powchainConfigBuilder.depositContractDeployBlock(
```

Comment:
Should this be `depositContractDeployBlockDefault`?

Reply:
Right! Good cactch 👍